src/main/host/shimipc.c

```diff
@@ -10,12 +10,12 @@
 
 // We use an int here because the option parsing library doesn't provide a way
 // to set a boolean flag to false explicitly.
-static gint _sendExplicitBlockMessage = true;
+static gint _useExplicitBlockMessage = true;
```

Comment:
While we're here, can we go ahead and change this to a `bool` option? That's how we're using it anyway so it's more appropriate. (I've been wishing it was a `bool` for a while.)

Reply:
The problem is that we can't change it to a bool without breaking the current behaviour. Right now `--send-explicit-block-message` defaults to true, so we'd either have to change the default to `false`, or rename the option to something like `--disable-explicit-block-message`.

The upcoming Rust changes will be removing all of this GLib stuff anyways, so I think it might be better to wait and make this breaking change with all of the other changes.

Reply:
Why can't we keep the default as `true`, and then use `G_OPTION_FLAG_REVERSE` the same way we do with `_use_object_counters`?

Reply:
> The upcoming Rust changes will be removing all of this GLib stuff anyways, so I think it might be better to wait and make this breaking change with all of the other changes.

OK, makes sense.

Reply:
> Why can't we keep the default as true, and then use G_OPTION_FLAG_REVERSE the same way we do with _use_object_counters?

Then we'd have to rename the option, otherwise the default would be true, and `--send-explicit-block-message` would also set it to true.

As we talked about a couple of weeks ago, since we want the the CLI options to override the yaml config options, in the Rust version all experimental bool options will have to take a true/false option (for example `--send-explicit-block-message true`) since if the yaml file specifies  `send-explicit-block-message: true`, you will need to be able to disable it on the command line. So rather than having two options (`--send-explicit-block-message` and `--disable-explicit-block-message`), we'll have a single `--send-explicit-block-message` option that takes either true or false.

Reply:
Got it, thanks!